package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "extension-activity-monitor",
+  "version": "1.0.0",
+  "description": "TBD",
+  "scripts": {
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "test:coveralls": "jest --coverage && coveralls < coverage/lcov.info",
+    "prettier:fix": "prettier --ignore-path ./.gitignore --write \"./**/*.{html,js,css}\"",
+    "prettier:check": "prettier ./.gitignore --check \"./**/*.{html,js,css}\"",
+    "lint:fix": "eslint --ignore-path ./.gitignore --fix \"./**/*.js\"",
+    "lint:check": "eslint --ignore-path ./.gitignore --quiet \"./**/*.js\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/extension-activity-monitor.git"
+  },
+  "keywords": [
+    "activityLog",
+    "extension activity monitor",
+    "firefox extension activity monitor",
+    "firefox",
+    "firefox extension logs",
+    "extension acitivty logs",
+    "mozilla"
+  ],
+  "author": "Mozilla <mozilla@mozilla.org>",
```

Comment:
I am not sure about it. Need your suggestion.

Reply:
I think we could set the author property of the package.json file to "Mozilla Add-ons Team", [as in the addons-linter repo](https://github.com/mozilla/addons-linter/blob/713cddc11ce64f9177040acb5f77f39176cb07ee/package.json#L35).

We could also use the same string for the author property in the manifest.json file.